notebooks/community/managed_notebooks/Fraud detection/Fraud Transaction Detection notebook.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! gcloud beta ai models upload \
</pre><p>Instead of using <code>gcloud</code>,  use Vertex SDK for Python. See samples here: https://github.com/googleapis/python-aiplatform/tree/main/samples/model-builder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>